spec/rubocop/cop/registry_spec.rb

```diff
@@ -172,6 +172,24 @@
     it 'exposes a list of cops' do
       expect(registry.cops).to eql(cops)
     end
+
+    context 'with cops having the same inner-most module' do
+      let(:cops) do
+        [
+          RuboCop::Cop::Foo::Bar,
+          RuboCop::Cop::Baz::Foo::Bar
+        ]
+      end
+
+      before do
+        stub_const('RuboCop::Cop::Baz::Foo::Bar', Class.new(RuboCop::Cop::Base))
+        stub_const('RuboCop::Cop::Foo::Bar', Class.new(RuboCop::Cop::Base))
```

Comment:
Absolute nitpick, but could you please switch these two lines, to be in the same order as lines 179–180?